Runtime/Model/JsonData/BacktraceAttributes.cs

```diff
@@ -66,7 +66,29 @@ public BacktraceAttributes(BacktraceReport report, Dictionary<string, string> cl
             SetMachineAttributes(onlyBuiltInAttributes);
             SetProcessAttributes(onlyBuiltInAttributes);
             SetSceneInformation(onlyBuiltInAttributes);
+#if UNITY_ANDROID
```

Comment:
Maybe we want to change this to a generic `ReadPlatformSpecificAttributes` function. Then we can get rid of this compile flag too.

Reply:
Moving this code to Android NativeClient